integration/kubernetes/k8s-pod-quota.bats

```diff
@@ -29,7 +31,8 @@ setup() {
 	sudo -E kubectl create -f "${pod_config_dir}/pod-quota-deployment.yaml"
 
 	# View information about the deployment
-	sudo -E kubectl get deployment "$deployment_name" --output=yaml | grep "forbidden: exceeded quota"
+	cmd="sudo -E kubectl get deployment "$deployment_name" --output=yaml | grep 'forbidden: exceeded quota'"
```

Comment:
- Nit: `grep -q` to avoid polluting the test output.
- Don't you need to quote the embedded double-quotes?
  ```sh
  cmd="sudo -E kubectl get deployment \"$deployment_name\" --output=yaml | grep 'forbidden: exceeded quota'"
  ```